Sources/Gtk/Utility/GObjectRepresentable.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Copyright © 2016 Tomas Linhart. All rights reserved.
+//
```

Comment:
No need to add the copyright to our code

Reply:
Ah that got copied over I guess, will remove.